examples/silicon_SJ_ECP/README

```diff
@@ -0,0 +1,19 @@
+Results:
+
+`./validate.sh ../examples/silicon_SJ_ECP/ 2 8 0.5 7`
+
+Shows a fit range if 2.5 bohr with polynomial of 5 to be best.
+
+`./analyse.sh ../examples/silicon_SJ_ECP 192 2.1608567484189534E+03 2.5 5` 
+
+Polynomial degree: 5
+Mean squared error: 0.00640, cross-validation MSE: 0.07056
+--> BEST
+----------------------------------------
+PCF           : 0.9555
+STD           : 0.0011
+----------------------------------------
+Lifetime (ps) : 225.9338 <-core-corrected
```

Comment:
On line 16, there is the text telling that the lifetime estimate has been core-corrected. On the level of the scripts and code, this means that as the script 'analyse.sh' in the 'scripts'-directory calls the python executable 'plyfit.py', it gives a value less than one (obtained from a two-component density functional theory calculation) to the argument 'corepart', as:

python ../positron_utils/pcf_fit/plyfit ... --corepart $COREPART_B15 ...

However, with the ECP pseudopotentials, the effect of the core is neglible. Thus, the corepart-argument should be set to 1 from within the 'analyse.sh'-script. Now the given value shows the actual lifetime estimate scaled down based on a corepart estimate from a calculation with AREP-pseudopotentals.

In conclusion, the existence of the README here is OK, but the values reported should be rewritten from analysis without core-corrections.

K.S.

Reply:
Yes I forgot. The settings in `analyse.sh` will probably need to be changed then. Maybe a if statement containing the different fractional contributions from each source with an extra option for "no core correction" i.e `--corepart 1`. 

See: 

https://github.com/simulak1/polyfit_to_positron_qmc_data/pull/1/commits/3a775e4a549cf841736bc0013fb4e6934a7d602f

Reply:
Suggestion for the changes to `analyse.sh` for choosing a core correction option in https://github.com/simulak1/polyfit_to_positron_qmc_data/pull/1/commits/f7f8fc4222e3b5af0e219f9d9043cbd655e2cf1e. I think some explanation of the core contributions and citations should also be added to the `README`.

Future changes can also be made to lines 104-116 in `analyse.sh`. Currently they are not being used.

Reply:
I agree, but propose we don't use if-statement and give only the corepart as the cmd-line option to plyfit.py.